libs/ledger-state/src/Cardano/Ledger/State/Query.hs

```diff
@@ -565,7 +565,8 @@ getDStateNoSharing dstateId = do
               iRTreasury = iRTreasury,
               deltaReserves = dStateIrDeltaReserves,
               deltaTreasury = dStateIrDeltaTreasury
-            }
+            },
+        dsDeposits = Map.empty -- FIXME, HELP ME FIX THIS
```

Comment:
What's the story here? We have to somehow populate this before we can get it? 
Do we need to fix this before merging?

Reply:
I think we can fix this in a follow up PR. This is needed for benchmarking the memory overhead of the ledger state, so need to block this PR on this problem.

What's needed for this is an extra table in the db schema and queries for storing and retrieving deposits from the sqlite db. 

@TimSheard Could you create a ticket about this? This would be a good task for @aniketd to pick up, since I know for sure he is familiar with persistent, so this would not be a hard task for him.

Reply:
This will depend on how we translate across the hard fork, so we will have to fill this in later. This whole module is for abstracting the ledgerstate from synching the chain. So this won't break anything for now.